benefits/core/templates/core/logged-out.html

```diff
@@ -7,12 +7,10 @@
 
 {% block main-content %}
   <div class="container">
-    <div class="row justify-content-lg-center">
+    <div class="row justify-content-center">
       <div class="col-md-6">
-        <h1 class="h2 text-center">
-          <span class="d-block pb-lg-8 pb-5">{% include "core/includes/icon.html" with name="happybus" %}</span>
```

Comment:
Putting a `span` with a `d-block` to wrap an `img`...... so weird!

Reply:
Yeah, weird also that the image/icon was _inside_ the `h1`. This makes way more sense :+1: